bus-mapping/src/evm/memory.rs

```diff
@@ -270,6 +270,11 @@ impl Memory {
             &self[addr..addr + MemoryAddress::from(32)],
         ))
     }
+
+    /// Returns the size of memory in words
+    pub fn size(&self) -> usize {
+        (self.0.len() + 31) / 32
```

Comment:
Maybe `self.0.len() >> 5` does the truncation.

Also, you can make the fn `const` or if not possible, at least make it `#[inline]`.

Reply:
>Maybe self.0.len() >> 5 does the truncation.

It is same as '/ 32', we need take care situations that divided by 32 left a reminder.

>Also, you can make the fn const or if not possible, at least make it #[inline].

`#[inline]` sound good.

Reply:
So if you need to take care when there's leftover in the division, there should be extra logic in the fn no?

As otherways you're purely just truncating.

Reply:
I think the memory size will always be multiple of 32.  Internally we represent the memory in bytes, but geth returns it as an array of words (where each word is 32 bytes)

Reply:
>geth returns it as an array of words (where each word is 32 bytes)

OK, in this case we could use `self.0.len() >> 5`.